etl/steps/data/garden/demography/2023-10-03/life_tables.meta.yml

```diff
@@ -0,0 +1,70 @@
+# NOTE: To learn more about the fields, hover over their names.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/dataset/
+dataset:
+  title: Life tables
+  update_period_days: 365
+  description: |-
+    Life tables from UN and HMD.
+
+    For period data, data prior to 1950 is from HMD, and data from 1950 onwards is from UN WPP.
+
+
+# Learn more about the available fields:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/tables/
+tables:
+  life_tables:
+    variables:
+      central_death_rate:
```

Comment:
I am not able to run this locally as I think the metadata is missing for `probability_of_survival` and `survivorship_ratio`. For me, they have the same titles in VariableMeta as a couple of the other variables. They are the last two in this screenshot: 
![Screenshot 2023-10-03 at 14 29 17](https://github.com/owid/etl/assets/10774728/ff28b230-0352-4698-a4f6-3cc0432c1231)

Reply:
Or somehow the metadata isn't propagating as expected, not sure.

Reply:
Might just be local to me as the CI is passing - have you been able to get the data onto staging @lucasrodes?